map/map.js

```diff
@@ -184,6 +199,166 @@ var DefineMap = Construct.extend("DefineMap",{
 		return this;
 	},
 	/**
+	 * @function can-define/map/map.prototype.assignDeep assignDeep
+	 * @parent can-define/map/map.prototype
+	 *
+	 * @description Sets multiple properties on a map instance or a property that wasn't predefined.
```

Comment:
what is meant by `wasn't predefined`

Reply:
I am guessing that "wasn't predefined" here indicates that there was no getter-setter added for that property yet (there was no definition on the prototype and the property wasn't in the constructing properties).